resources/function_help/json/array_concat

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "array_concat",
```

Comment:
This is differing from Postgres' naming ("array_cat"). Not sure if that's an issue or not - personally I prefer your "array_concat".